src/internal/_arrayChainRec.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (function() {
+  function stepNext(x) { return {value: x, done: false }; }
+  function stepDone(x) { return {value: x, done: true }; }
+
+  return function _chainRec(f, i) {
```

Comment:
`i` stands for `initial`, also it's used in fantasy land spec.
For that reason I think it's better to leave it as is.